pages/login/login.html

```diff
@@ -1,69 +1,73 @@
+    <link
+      rel="stylesheet"
+      href="https://starui-component-library.netlify.app/index.css"
+    />
+    <link rel="stylesheet" href="./login.css" />
+    <link rel="stylesheet" href="../../index.css" />
+    <title>eCommerce | Login</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
+        <h2><a href="/">FootballHQ</a></h2>
+      </div>
 
-    <form class="form">
-      <input type='text' class="form-input small-text" placeholder="Enter Product">
-    </form> 
-                                                                                                                    
-    <div class="right-nav">
-      <button class="btn primary-btn">Login</button>
-      <ul>                                                    
-        <li>
-          <div class="badge">
-            <i class="fas fa-heart fa-2x"></i>
-            <div class="badge-number">1</div>
-          </div>
-        </li>
-        <li>
-          <div class="badge">
-            <i class="fas fa-shopping-cart fa-2x"></i>
-            <div class="badge-number">1</div>
-          </div>
-        </li>                                 
-      </ul>
-    </div>
-    <button class="navbar-btn">
-      <span class="hamburger"></span>
-    </button>
-  </nav>
-  <div class="login-container">
-    <div class="login-card">
-      <h2>Login</h2>
-      <form>
-        <label>
-          Email Address
-        </label>
-        <input class="form-input" type="text">
-        <label>
-          Password
-        </label>
-        <input class="form-input" type="text">
-        <div class="help-container">
-          <input type="radio">
-          <label>Remember Me ?</label>
-          <a>Forgot your Password </a>
-        </div>
-        <button class="btn primary-btn" type="submit">Login</button>
-        <p><a>Create New Account ?</a></p>
+      <form class="form">
+        <input
+          type="text"
+          class="form-input small-text"
+          placeholder="Enter Product"
+        />
       </form>
+
+      <div class="right-nav">
+        <button class="btn primary-btn">Login</button>
+        <ul>
+          <li>
+            <div class="badge">
+              <i class="fas fa-heart fa-2x"></i>
+              <div class="badge-number">1</div>
+            </div>
+          </li>
+          <li>
+            <div class="badge">
+              <i class="fas fa-shopping-cart fa-2x"></i>
+              <div class="badge-number">1</div>
+            </div>
+          </li>
+        </ul>
+      </div>
+      <button class="navbar-btn">
+        <span class="hamburger"></span>
+      </button>
+    </nav>
+    <div class="login-container">
+      <div class="login-card">
+        <h2>Login</h2>
+        <form>
+          <label> Email Address </label>
+          <input class="form-input" type="text" />
+          <label> Password </label>
+          <input class="form-input" type="text" />
+          <div class="help-container">
```

Comment:
You can provide `id` and `for` attribute to input and label elements respectively.